spec/features/course/achievement_management_spec.rb

```diff
@@ -13,123 +11,205 @@
       login_as(user, scope: :user)
     end
 
-    describe 'achievement creation' do
-      before { visit new_course_achievement_path(course) }
-      subject { click_button I18n.t('helpers.submit.achievement.create') }
-
-      context 'with invalid information' do
-        before { subject }
-
-        it 'stays on the same page' do
-          expect(page).to have_button('helpers.submit.achievement.create')
-        end
-
-        it 'shows errors' do
-          expect(page).to have_css('div.has-error')
-        end
-      end
-
-      context 'with valid information' do
-        let(:achievement) { build(:course_achievement, course: course) }
+    context 'As a User' do
 
+      describe 'I can create an achievement' do
```

Comment:
'scenario' is the user story, so this line is `scenario 'I can create an achievement'`.